torch/nn/utils/parametrize.py

```diff
@@ -701,8 +699,9 @@ def type_before_parametrizations(module: Module) -> type:
 def transfer_parametrizations_and_params(
     from_module: Module, to_module: Module, tensor_name: Optional[str] = None
 ) -> Module:
-    r"""Transfers parametrizations and the parameters they parametrize from from_module
-    to to_module. If tensor_name is specified, only transfers the specified parameter, otherwise
+    r"""Transfer parametrizations and the parameters they parametrize from from_module to to_module.
```

Comment:
Preexisting, but from_module and to_module, and tensor_name in the next line should be `:attr:`